app/view_models/verify/base.rb

```diff
@@ -0,0 +1,88 @@
+module Verify
+  class Base
+    include Rails.application.routes.url_helpers
+
+    def initialize(error: nil, remaining_attempts:)
+      @error = error
+      @remaining_attempts = remaining_attempts
+    end
+
+    attr_reader :error, :remaining_attempts
+
+    def title
+      I18n.t("idv.titles.#{step_name}")
+    end
+
+    def modal_partial
+      if error.present?
+        'shared/modal_verification'
+      else
+        'shared/null'
+      end
+    end
+
+    def warning_partial
+      if error == 'warning'
+        'shared/modal_verification_warning'
+      else
+        'shared/null'
+      end
+    end
+
+    def message
+      html_paragraph(text: I18n.t("idv.modal.financials.#{error}")) if error
```

Comment:
great catch. fixed at 9fc8d602d264c32afe892592077c27e0b542ee45

I went through to manually check these errors, but somehow missed that the translations were showing the wrong message